### PR TITLE
Update last_renewed for some clients

### PR DESCRIPTION
**Pull request by bodqhrohro**
I used evidence like last commit/push time, or last release date, for proprietary software.

### CODE REVIEW

File: data/clients.json [hunk 1]
```diff
@@ -128,7 +128,7 @@
     },
     {
         "doap": null,
-        "last_renewed": "2020-05-13T19:46:58",
+        "last_renewed": "2021-01-18T17:43:00",
         "name": "ChatSecure",
```
Comment (Echolon) [inline data/clients.json:132]: Let's not be too enthusiastic, I am following the iOS developments and I know this is not actively being developed anymore (unfortunately). I would expect the developer to update preferably, if he reconsiders to start over.

File: data/clients.json [hunk 2]
```diff
@@ -147,7 +147,7 @@
     },
     {
         "doap": null,
-        "last_renewed": null,
+        "last_renewed": "2013-04-25T00:00:00",
```
Comment (Echolon) [inline data/clients.json:150]: The idea is not to simply list all clients. Ideally, the developer update here themselves. Hence, I would not like to update this here.

File: data/clients.json [hunk 3]
```diff
@@ -208,14 +208,14 @@
...
-            "Other",
+            "OS/2",
             "Windows"
         ],
         "url": "https://eyecu.opiums.eu"
     },
     {
         "doap": null,
-        "last_renewed": null,
+        "last_renewed": "2021-03-03T05:30:05",
```
Comment (Echolon) [inline data/clients.json:218]: Maybe this is getting a bit emotional, but I don't think we should put efforts in placing Pidgin here. The XMPP implementation is really old and as long as the maintainer comes here, I would refuse to update this.

File: data/clients.json (hunk 2 above)
Comment (Echolon) [reply data/clients.json:150]: Sorry, I just see you are setting old time stamps - sorry for that!

File: data/clients.json [hunk 4]
```diff
@@ -252,13 +252,15 @@
     },
     {
         "doap": null,
-        "last_renewed": "2019-07-16T20:02:35",
+        "last_renewed": "2021-01-12T00:00:00",
```
Comment (Echolon) [inline data/clients.json:255]: It doesn't even support XMPP anymore. Let's remove it entirely please. It is also know as scammer/bloat software.

File: data/clients.json [hunk 5]
```diff
@@ -321,7 +323,7 @@
     },
     {
         "doap": null,
-        "last_renewed": "2018-08-29T15:00:00",
+        "last_renewed": "2021-05-20T18:25:00",
```
Comment (Echolon) [inline data/clients.json:326]: I'm pretty sure it is not developed anymore: https://github.com/jitsi/jitsi/releases/tag/Jitsi-2.10

File: data/clients.json [hunk 6]
```diff
@@ -374,7 +376,7 @@
     },
     {
         "doap": null,
-        "last_renewed": "2019-07-16T20:18:31",
+        "last_renewed": "2021-05-21T00:00:00",
```
Comment (Echolon) [inline data/clients.json:379]: I'm having doubt here, too

File: data/clients.json [hunk 7]
```diff
@@ -444,7 +449,7 @@
     },
     {
         "doap": null,
-        "last_renewed": "2019-07-16T20:26:34",
+        "last_renewed": "2021-05-18T07:08:18",
```
Comment (Echolon) [inline data/clients.json:452]: I'm having doubt here, too, see my comment on Finch

File: data/clients.json [hunk 8]
```diff
@@ -737,7 +742,7 @@
     },
     {
         "doap": null,
-        "last_renewed": "2019-07-16T20:55:15",
+        "last_renewed": "2021-02-09T00:00:00",
```
Comment (Echolon) [inline data/clients.json:745]: Please remove it entirely: https://github.com/zom/Zom-iOS-XMPP
It not even using XMPP anymore.

**Comment by Echolon** [comment]
@bodqhrohro thanks for doing a lot of efforts! I still have some change request. I hope you agree on this!

**Review by Echolon — CHANGES_REQUESTED** [review]
@bodqhrohro thanks for doing a lot of efforts! I still have some change request. I hope you agree on this!

**Comment by guusdk** [comment]
Although I appreciate the effort, the XSF board decided on a policy where only people associated to the projects should be allowed to add/renew the corresponding listing (for the record: I didn't agree with that, but still, it is policy).

**Comment by Echolon** [comment]
Ah okay, then I have to close this unfortunately.

Evtl. it is possible to talk to the relevant developers of at least the developed clients?

File: data/clients.json (hunk 3 above)
Comment (bodqhrohro) [reply data/clients.json:218]: > The XMPP implementation is really old

... and still the best among what Pidgin supports out-of-box at all. The implementations for IRC and Bonjour are way more primitive, and the rest of prpls seem to be just a pile of dead code for dead protocols.

If Pidgin is not an XMPP-first client anymore, then what is that, just a framework for great third-party prpls? ;)

> as the maintainer comes here

Should I notify them?

Comment (Echolon) [reply data/clients.json:218]: @bodqhrohro - I recommend to do the following: 

Reach out the developer of interest and ask to update the timestamp **IF**

- they are still actively developing and maintaining their project / XMPP implementation of their project
- think it is worth to guide new people to their project

Comment (Echolon) [reply data/clients.json:218]: For all the stale projects we might create a separate archive collection

**Comment by grimmy** [comment]
fwiw, pidgin is aware and okay not being on this list until we get our xmpp implementation to a minimum of the modern xmpp spec.

**Comment by Echolon** [comment]
> fwiw, pidgin is aware and okay not being on this list until we get our xmpp implementation to a minimum of the modern xmpp spec.

Alright - thank you for the feedback! We will not update as long as the developers does it

**Pull request closed without merging**